librz/core/rop.c

```diff
@@ -394,6 +398,29 @@ RZ_IPI RzRopRegInfo *rz_core_rop_reg_info_dup(RzRopRegInfo *src) {
 	return dup;
 }
 
+/**
+ * \brief Find the ROP Register information for the given register
+ * \param gadget_info Pointer to the RzRopGadgetInfo
+ * \param name Register to filter the dependencies
+ *
+ * \return List of register rop information
+ */
+RZ_API RZ_NULLABLE RzList /*<RzRopRegInfo *>*/ *rz_core_rop_reg_info_find(const RZ_NONNULL RzRopGadgetInfo *gadget_info, const RZ_NONNULL char *name) {
```

Comment:
Missing ownership of the return value

Reply:
https://github.com/rizinorg/rizin/pull/4582#discussion_r1730545228